play/apps/tournament/views/tournament.py

```diff
@@ -39,16 +39,24 @@ def new(request):
 @admin_required
 @login_required
 def edit(request, id):
-    tournament_group = Tournament.objects.get(id=id)
+    tournament = Tournament.objects.get(id=id)
 
     if request.method == 'POST':
-        form = TournamentForm(request.POST, instance=tournament_group)
+        form = TournamentForm(request.POST, instance=tournament)
 
         if form.is_valid():
             form.save()
-            messages.success(request, f'Tournament group "{tournament_group.name}" updated')
+            messages.success(request, f'Tournament "{tournament.name}" updated')
             return redirect('/tournaments/')
     else:
-        form = TournamentForm(instance=tournament_group)
+        form = TournamentForm(instance=tournament)
 
     return render(request, 'tournament/edit.html', { 'form': form })
+
+
+@admin_required
+@login_required
+def delete(request, id):
+    tournament = Tournament.objects.get(id=id)
```

Comment:
You could just call `.delete` here. Also, would be nice to send a success `message` that gets displayed in the UI after deleting the tournament